product_docs/docs/pgd/5/reference/catalogs-internal.mdx

```diff
@@ -197,5 +200,4 @@ An internal state table storing the type of each non-local sequence. We recommen
 | Name    | Type | Description                                                 |
 | ------- | ---- | ----------------------------------------------------------- |
 | seqid   | oid  | Internal OID of the sequence                                |
-| seqkind | char | Internal sequence kind ('l'=local,'t'=timeshard,'s'=snowflakeid,'g'=galloc) |
-
+| seqkind | char | Internal sequence kind (`'l'`=local,`'t'`=timeshard,`'s'`=snowflakeid,`'g'`=galloc) |
```

Comment:
@djw-m provided an example of how this can look: 
```
bdrdb=> select * from bdr.sequence_kind;
       seqid       | seqkind
-------------------+---------
 seqtest_mykey_seq | g
(1 row)
```

Since the g is not surrounded by apostrophes in the output, I don't think we need to nestle the g in apostrophes and backticks for the emphasis. I think backticks are enough here. WDYT? 

Or maybe there is a different reason why this makes sense @ebgitelman ?

Reply:
Sometimes single quotes are needed. That's how the original was written. If that is not the case, then I will remove the apostrophes.

Reply:
single quotes are needed when setting the value but as the field is a char it should be evident that single quotes are needed to delimit the char when setting it. The actual value is l/t/s/g and that's really what we want to see in the reference.